plugins/services/src/js/components/modals/NewCreateServiceModal.js

```diff
@@ -123,7 +260,8 @@ class NewServiceFormModal extends Component {
     }
 
     // Close if picker is open, or if editing a service in the form
-    if (servicePickerActive || (!serviceReviewActive && this.props.isEdit)) {
+    if (servicePickerActive ||
+      (!serviceReviewActive && location.pathname.includes('/edit'))) {
```

Comment:
wonder if it's possible to forge such an app name so that it will give false positives here 🤔